opentelemetry-exporter-cloud-monitoring/tests/test_cloud_monitoring.py

```diff
@@ -1,3 +1,4 @@
+# pylint: disable=too-many-statements
```

Comment:
Why is this needed? And can you resolve this without resorting to disabling tests?

Reply:
not sure honestly, but I wouldn't consider this removing tests, more like removing a lint exception that I don't believe matters